CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Changelog
 
 ### 4.0.0 (Next)
+* [#176](https://github.com/alexa-js/alexa-app/pull/176): Added flexibility to attach alexa-app to express app or router - [@bgjehu](https://github.com/bgjehu).
```

Comment:
@dblock I don't know how you guys handle the release. If the changes is going in to 4.0.0, it shouldn't break the alexa-app-server for alexa-app-server's package.json is using alexa-app ^3.0.0
and it is good idea to throw exception if both specified! Thank you!